nolibc/src/x86_64/syscall.rs

```diff
@@ -2,11 +2,18 @@
 
 //! syscall type and constants
 
-#![allow(missing_docs)]
+// Number values generated with:
+//
+// ```
+// bindgen /usr/include/sys/syscall.h \
+//   | sed -rn 's|pub const SYS_([a-z][a-z0-9_]*): u32 = ([0-9]+);|\1 = \2,|p' \
+//   | tr [:lower:] [:upper:] \
+//   | sort -g -t= -k2
+// ```
 
 enumerate::enumerate! {
     #[derive(Copy, Clone)]
-    pub enum SysCall: u64 {
+    pub enum Number: u64 {
```

Comment:
I liked the name `SysCall` better here. Client code will have to alias this module themselves like the enarx-sgx-keep-shim does in this PR or do something like `syscall::Number::EXIT` if they prefer to type out full namespaces with this change.

edit: please see top-level review comment before addressing this